src/Controls/src/Core/DependencyAttribute.cs

```diff
@@ -7,11 +8,13 @@ namespace Microsoft.Maui.Controls
 	public sealed class DependencyAttribute : Attribute
 	{
 		/// <include file="../../docs/Microsoft.Maui.Controls/DependencyAttribute.xml" path="//Member[@MemberName='.ctor']/Docs" />
-		public DependencyAttribute(Type implementorType)
+		public DependencyAttribute(
+			[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicParameterlessConstructor)] Type implementorType)
```

Comment:
Would this one potentially need to keep interfaces, too?

Like in this test example:

https://github.com/dotnet/maui/blob/18e0f4ebbcca7c904ccdb67e6439cb72382e2a40/src/Controls/tests/Core.UnitTests/DependencyServiceTests.cs#L10-L29

Maybe we can save that for a later PR.